api/applications/views.py

```diff
@@ -50,7 +50,7 @@ class ApplicationList(JSONAPIBaseView, generics.ListCreateAPIView, ODMFilterMixi
     view_category = 'applications'
     view_name = 'application-list'
 
-    renderer_classes = [renderers.JSONRenderer]  # Hide from web-browsable API tool
+    renderer_classes = [JSONRendererWithESISupport, JSONAPIRenderer, ]  # Hide from web-browsable API tool
```

Comment:
This doesn't hide the endpoint (because at minimum it has to respond to JSON requests)... it just makes it less user-friendly.

If we move to swagger, I agree that will likely ignore any custom rendering settings we have in place, in which case the value of custom behavior for this class goes way down.